esp/templates/program/newprogram_review.html

```diff
@@ -38,18 +38,21 @@ <h2>Program Creation Review</h2>
 <div id="catalog">
 <table align="center">
 <tr>
-<th>The following program modules will be added:</th>
+<th colspan="2">The following program modules will be added:</th>
 </tr>
-<tr><th class="materials">Module Name</th></tr>
+<tr><th class="materials" colspan="2">Module Name</th></tr>
 {% for item in modules %}
-    <tr><td>{{ item.0 }}</td></tr>
+    {% if forloop.counter0|divisibleby:"2" %}<tr><td style="text-align:center">
+    {% else %} <td style="text-align:center">{% endif %}
+    {% for word in item.0.split %}{% if word != "Program" and word != "Module:" %}{{word}}{% endif %}{% if not forloop.last %} {% endif %}{% endfor %}</td>
+    {% if forloop.last or not forloop.counter0|divisibleby:"2" %}</tr>{%endif %}
 {% endfor %}
 </table>
 </div>
 </p>
 
 <center>
-<div class="info"><b>Warning</b>: it is difficult to undo the changes made by this function.  If you would like to change anything, please go back and re-submit.</div>
+<div class="info"><b>If you would like to change anything, please <a href="javascript:history.go(-1);" title="Go back">go back</a> and re-submit.</div>
```

Comment:
I would make sure to keep the "there is no undo button"!  Although that may be slightly less true than when written.

Reply:
I can add a warning, but everything on this page can be undone via /admin/program/ at this point

Reply:
Ok, I guess it's fine to remove then.